test/Microsoft.Windows.CsWin32.Tests/GeneratorTests.cs

```diff
@@ -1284,127 +1045,9 @@ public async Task TestSimpleMethod()
-	using win32 = global::Windows.Win32;
-
-	namespace Foundation
-	{
-		[DebuggerDisplay(""{Value}"")]
-		internal readonly partial struct HWND
-			: IEquatable<HWND>
-		{
-			internal readonly nint Value;
-			internal HWND(nint value) => this.Value = value;
-			public static implicit operator nint(HWND value) => value.Value;
-			public static explicit operator HWND(nint value) => new HWND(value);
-			public static bool operator ==(HWND left, HWND right) => left.Value == right.Value;
-			public static bool operator !=(HWND left, HWND right) => !(left == right);
-
-			public bool Equals(HWND other) => this.Value == other.Value;
-
-			public override bool Equals(object obj) => obj is HWND other && this.Equals(other);
-
-			public override int GetHashCode() => this.Value.GetHashCode();
-		}
-	}
-}
-".Replace("\r\n", "\n")),
-                    (typeof(SourceGenerator), "PInvoke.User32.g.cs", @"// ------------------------------------------------------------------------------
-// <auto-generated>
-//     This code was generated by a tool.
-//
-//     Changes to this file may cause incorrect behavior and will be lost if
-//     the code is regenerated.
-// </auto-generated>
-// ------------------------------------------------------------------------------
-
-#pragma warning disable CS1591,CS1573,CS0465,CS0649,CS8019,CS1570,CS1584,CS1658
-namespace Windows.Win32
-{
-	using global::System;
-	using global::System.Diagnostics;
-	using global::System.Runtime.CompilerServices;
-	using global::System.Runtime.InteropServices;
-	using win32 = global::Windows.Win32;
-
-
-	/// <content>
-	/// Contains extern methods from ""User32.dll"".
-	/// </content>
-	internal static partial class PInvoke
-	{
-		/// <summary>The ReleaseDC function releases a device context (DC), freeing it for use by other applications. The effect of the ReleaseDC function depends on the type of DC. It frees only common and window DCs. It has no effect on class or private DCs.</summary>
-		/// <param name=""hWnd"">A handle to the window whose DC is to be released.</param>
-		/// <param name=""hDC"">A handle to the DC to be released.</param>
-		/// <returns>
-		/// <para>The return value indicates whether the DC was released. If the DC was released, the return value is 1. If the DC was not released, the return value is zero.</para>
-		/// </returns>
-		/// <remarks>
-		/// <para><see href=""https://docs.microsoft.com/windows/win32/api//winuser/nf-winuser-releasedc"">Learn more about this API from docs.microsoft.com.</see></para>
-		/// </remarks>
-		[DllImport(""User32"", ExactSpelling = true)]
-		[DefaultDllImportSearchPaths(DllImportSearchPath.System32)]
-		internal static extern int ReleaseDC(win32.Foundation.HWND hWnd, win32.Graphics.Gdi.HDC hDC);
-	}
-}
-".Replace("\r\n", "\n")),
+                    LoadExpectedOutput("HDC.g.cs"),
+                    LoadExpectedOutput("HWND.g.cs"),
+                    LoadExpectedOutput("PInvoke.User32.g.cs"),
```

Comment:
📝 These files are the highest risk of churn. This one will work as long as _either_ no other method from user32.dll is added to NativeMethods.txt _or_ the test itself is updated to include more methods than just ReleaseDC in the test.